api/metrics/multi_gatherer_test.go

```diff
@@ -4,13 +4,17 @@
 package metrics
 
 import (
+	"errors"
 	"testing"
 
+	"github.com/prometheus/client_golang/prometheus"
 	"github.com/stretchr/testify/require"
 
 	dto "github.com/prometheus/client_model/go"
 )
 
+var errTest = errors.New("non-nil error")
```

Comment:
unused local variable.

Reply:
Hmmm - this variable is used here: https://github.com/ava-labs/avalanchego/blob/1b4bb45e3b4aaba3f8fd7860b6835bd2de3fc5f8/api/metrics/multi_gatherer_test.go#L42-L56 But it probably makes sense to change this from a package var to a function var... I'll update that